packages/flutter_test/lib/src/controller.dart

```diff
@@ -2221,12 +2222,17 @@ abstract class WidgetController {
     int maxIteration = 50,
     Duration duration = const Duration(milliseconds: 50),
   }) {
+    assert(moveStep != Offset.zero);
+    assert(maxIteration > 0);
```

Comment:
(from offline review with @Piinks) these asserts (plus the delta assert) look valid. Are they related to this change specifically or just good to have generally? And can we add tests for them in either case?

Reply:
Should I revert the asserts and the corresponding tests first, and then submit a new PR later?